Dockerfile

```diff
@@ -18,8 +18,11 @@ FROM centos:7
 RUN yum install -y java-1.8.0-openjdk-devel
 ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
 
-ARG TEST_JAR_VERSION
+ENV HADOOP_VERSION 3.1.1
```

Comment:
Seems like this should be an argument

Reply:
Or could have the maven dependency copy strip the versions.  Then dockerfile does not need to know versions.

Reply:
cservice

Reply:
Fixed now